functions/utils/application.js

```diff
@@ -0,0 +1,44 @@
+const createAppObject = (body) => {
+  const appObj = {
+    title: body["title"],
+    email: body["email"],
+  };
+  return appObj;
+};
+
+const validateAppData = (data) => {
+  const fields = [];
+  let errors = [];
+
+  fields.forEach((key) => {
+    switch (key) {
+      case "email":
+        if (data[key] === "") {
+          errors.push("Email is Invalid");
+        }
+        /*
+                validate anything necessary for data[key]
+                errors.push("Error")
```

Comment:
can't wait to see how long this switch statement will be

Reply:
It will be as long as there fields in the application object that is created. I'm doing it like this so it hopefully keeps code cleans when we have a lot of conditionals.

Reply:
makes sense. Recalling how long that list is for applicant info that needs to be validated, let me know if you want to divvy up some of that work!

Reply:
That will be done in a later sprint once the finalized schema is ready so if that's in your interest, you can definitely work on it!